### PR TITLE
[Platform] Add Ingrasys S9130-32X and S9230-64X with Nephos Switch ASIC

### PR DESCRIPTION
**Pull request by kaiyu22**
**- What I did**
Add switch ASIC vendor: Nephos
Add Nephos platforms: Ingrasys S9130-32X, Ingrasys S9230-64X

**- How I did it**
Add platform/nephos files
Add platform/nephos/sonic-platform-modules-ingrasys submodule
Add device/ingrasys/x86_64-ingrasys_s9130_32x-r0 files
Add device/ingrasys/x86_64-ingrasys_s9230_64x-r0 files
Add SONiC to support Nephos platform

**- How to verify it**
To build SONiC installer image and docker images, run the following commands:

- make configure PLATFORM=nephos
- make target/sonic-nephos.bin

Check system and network feature is worked as well

**- Description for the changelog**
Add switch ASIC vendor and platforms for Nephos

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: platform/nephos/python-saithrift.mk [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# python-saithrift package
+
+PYTHON_SAITHRIFT_NEPHOS = python-saithrift_0.9.4_amd64.deb
```
Comment (lguohan) [inline platform/nephos/python-saithrift.mk]: why still 0.9.4?

File: platform/nephos/libsaithrift-dev.mk [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# libsaithrift-dev package
+
+LIBSAITHRIFT_DEV_NEPHOS = libsaithrift-dev_0.9.4_amd64.deb
```
Comment (lguohan) [inline platform/nephos/libsaithrift-dev.mk]: why still 0.9.4?

File: platform/nephos/python-saithrift.mk (hunk 1 above)
Comment (kaiyu22) [reply platform/nephos/python-saithrift.mk]: @lguohan , we followed [Broadcom platform](https://github.com/Azure/sonic-buildimage/blob/master/platform/broadcom/python-saithrift.mk) and saw that each platform vendor using 0.9.4 in this file as well.

File: platform/nephos/libsaithrift-dev.mk (hunk 2 above)
Comment (kaiyu22) [reply platform/nephos/libsaithrift-dev.mk]: @lguohan , we followed [Broadcom platform](https://github.com/Azure/sonic-buildimage/blob/master/platform/broadcom/libsaithrift-dev.mk) and saw that each platform vendor using 0.9.4 in this file as well.

**Comment by lguohan** [comment]
I think you need some changes in this 

https://github.com/Azure/sonic-sairedis/blob/master/syncd/scripts/syncd_init_common.sh

I do not really see you change that, how can your syncd get started?

File: platform/nephos/python-saithrift.mk (hunk 1 above)
Comment (lguohan) [reply platform/nephos/python-saithrift.mk]: ok. let's keep as it is for now. We'll revisit this issue again to upgrade the saithrift to 1.2.

**Comment by kaiyu22** [comment]
I will send PR to sonic-sairedis later.

**Comment by lguohan** [comment]
I'll wait for that PR before merge this one since this one may depends on that one.

**Comment by kaiyu22** [comment]
The [PR#271 of sonic-sairedis](https://github.com/Azure/sonic-sairedis/pull/271) has been created, thanks for your review.

**Comment by lguohan** [comment]
thanks. can you advance the syncd submodule in this PR to include that PR?

**Comment by kaiyu22** [comment]
Add a commit to advance sonic-sairedis submodule to include [sonic-sairedis#271 (Add Nephos ASIC)](https://github.com/Azure/sonic-sairedis/pull/271)

**Pull request merged**